indico/core/db/sqlalchemy/locations.py

```diff
@@ -189,3 +189,44 @@ def location_data(self):
             return {'source': data_source, 'room': data_source.room, 'room_name': data_source.room_name,
                     'venue_name': data_source.venue_name, 'address': data_source.address,
                     'inheriting': self.inherit_location}
+
+    @property
+    def location_data_for_UI(self):
```

Comment:
I really need a better name for it but can't come up with something. Think is that I need the `room_id` in the representation of the `location_data` in the template, and not the `room` itself.